src/MiniCover.HitServices/HitService.cs

```diff
@@ -1,73 +1,14 @@
-﻿using System;
-using System.Collections.Concurrent;
-using System.IO;
-
-namespace MiniCover.HitServices
+﻿namespace MiniCover.HitServices
 {
     public static class HitService
     {
-        public static MethodContext EnterMethod(
+        public static MethodScope EnterMethod(
             string hitsPath,
             string assemblyName,
             string className,
             string methodName)
         {
-            return new MethodContext(hitsPath, assemblyName, className, methodName);
-        }
-
-        public class MethodContext : IDisposable
```

Comment:
Renamed this to MethodScope and moved into a separate file. Because of this renaming the IL representation in all tests changed from `MiniCover.HitServices.HitService/MethodContext` to  `MiniCover.HitServices.MethodScope`